src/main/java/org/mockito/session/MockitoSessionBuilder.java

```diff
@@ -55,6 +85,21 @@
     @Incubating
     MockitoSessionBuilder strictness(Strictness strictness);
 
+    /**
+     * Configures logger used by {@code MockitoSession} for emitting hints when finishing
+     * the session.
+     * See examples in {@link MockitoSession}.
+     *
+     * @param logger for hints emitted when finishing {@code MockitoSession}.
```

Comment:
How about we explain to the user when to use this method? We can say that we recommend to use strict stubs rather than warnings, and that this method is used by framework integrations to keep backwards compatibility with early versions of Mockito 2.x

Reply:
Sure!